src/udev/udevadm-control.c

```diff
@@ -59,7 +59,7 @@ int control_main(int argc, char *argv[], void *userdata) {
 
         static const struct option options[] = {
                 { "exit",             no_argument,       NULL, 'e'      },
-                { "log-level",        required_argument, NULL, 'l'      },
+                { "log-level",        optional_argument, NULL, 'l'      },
```

Comment:
hmm, `optional_argument` sucks in getopt(), to the point it's simply not usable.

i.e. it's not clear what this means:

```c
foobar --somearg qqq
```

as long as `--somearg` is set to be a `required_argument`, then `qqq` will be the optarg for --somearg, and not a positional argument. But if it is `optional_argument`, then it might or might not be a positional argument. who knows...

Hence, in our codebase we don't want to see new uses of `optional_argument`.

please use something different, for example simply show the current value if the empty string string is passed.